geostore/Dockerfile

```diff
@@ -1,26 +1,22 @@
-FROM ubuntu:22.04 as build
-
-COPY geostore/pip.txt geostore/poetry.txt poetry.lock poetry.toml pyproject.toml /opt/
-WORKDIR /opt
-RUN apt-get update \
-    && apt-get install --assume-yes --no-install-recommends \
-    curl python-is-python3 python3-pip \
-    && rm -rf /var/lib/apt/lists/* \
-    && pip install --no-cache-dir --requirement=pip.txt \
-    && pip install --no-cache-dir --requirement=poetry.txt
+FROM python:3.9-slim as build
+
+WORKDIR /opt/
+
 ARG task
-RUN poetry install --extras=${task} --no-root --only=main
+ARG packaging
 
+RUN python -m venv /opt/.venv
+ENV PATH="/opt/.venv/bin:$PATH"
```

Comment:
I think this is only needed so that the `pip` command below runs the right one. So changing that line to `RUN /opt/.venv/bin/pip […]` would be consistent with `ENTRYPOINT` below, and would remove the need for this line.